alfalfa_worker/Dockerfile

```diff
@@ -173,19 +173,15 @@ RUN git clone --branch $PYFMI_TAG --depth 1 https://github.com/modelon-community
     && python setup.py install \
     && cd .. && rm -rf PyFMI
 
-
-# There are some tweaks because Ubuntu 20.04 has a PEP404 error with the version.
-# See https://bugs.launchpad.net/ubuntu/+source/python-debian/+bug/1926870
-# The below installs poetry 1.1.15 because 1.2.1 has the error.
-# It then copies over the poetry files and installs the dependencies with --no-dev
-RUN pip3 install poetry==1.1.15
+# Install poetry
+RUN pip3 install poetry==1.3.1
 
 # Copy and install poetry-based dependencies
 COPY pyproject.toml $BUILD_DIR
 COPY poetry.lock $BUILD_DIR
 
-RUN poetry config virtualenvs.create false \
-    && poetry install --no-dev
+# Install 'production' dependencies from poetry
+RUN poetry install --no-dev
```

Comment:
pretty big difference here now that we aren't disabling virtualenvs. Python that is used is now in a Poetry virtualenv. The "start-worker" script adds the virtualenv path to the system path to make everything work.